spec/models/response_set_spec.rb

```diff
@@ -543,32 +543,32 @@ def generate_responses(count, mandatory = nil, dependent = nil, triggered = nil)
     @response_set.responses << FactoryBot.create(:response, :string_value => "pecan pie", :question_id => @what_flavor.id, :answer_id => @what_flavor.answers.first.id, :response_set_id => @response_set.id)
   end
   it "should export a string with responses" do
-    @response_set.responses.size.should == 2
+    expect(@response_set.responses.size).to eq(2)
     csv = @response_set.to_csv
-    csv.is_a?(String).should be_true
-    csv.should match "question.short_text"
-    csv.should match "What flavor?"
-    csv.should match /pecan pie/
+    expect(csv.is_a?(String)).to be(true)
+    expect(csv).to match "question.short_text"
+    expect(csv).to match "What flavor?"
+    expect(csv).to match /pecan pie/
   end
 end
 
-describe ResponseSet, "#as_json" do
+describe ResponseSet, "#as_json", type: :model do
   let(:rs) {
     FactoryBot.create(:response_set, :responses => [
-          FactoryBot.create(:response, :question => FactoryBot.create(:question), :answer => FactoryBot.create(:answer, :response_class => :string), :string_value => '2')])
+    FactoryBot.create(:response, :question => FactoryBot.create(:question), :answer => FactoryBot.create(:answer, :response_class => :string), :string_value => '2')])
   }
 
   let(:js) {rs.as_json}
```

Comment:
Actually -- should this whole thing be removed since we removed `as_json` from this model?